src/main/java/uk/gov/hmcts/ccd/data/caseaccess/CaseLinkEntity.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.ccd.data.caseaccess;
+
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "case_link")
+public class CaseLinkEntity {
+
+    public static class CaseLinkPrimaryKey implements Serializable {
+        @Column(name = "case_id", nullable = false)
+        private Long caseId;
+        @Column(name = "linked_case_id", nullable = false)
+        private Long linkedCaseId;
```

Comment:
The case data is store in a table called `case_data`.  All other tables that link to this property use `case_data_id`  and properties called `caseDataId`, so should this.

Reply:
Ignore comment, keep name matching field.